tests/test_mlkit.py

```diff
@@ -277,7 +277,11 @@
 PAGE_SIZE_VALUE_ERROR_MSG = 'Page size must be a positive integer between ' \
                             '1 and {0}'.format(mlkit._MAX_PAGE_SIZE)
 INVALID_STRING_OR_NONE_ARGS = [0, -1, 4.2, 0x10, False, list(), dict()]
-
+PUBLISH_AND_UNPUBLISH_VALUE_ARGS = [
```

Comment:
Instead of putting these in the module namespace, you might want to put them in the `TestPublishUnpublish` class.

Same goes for most other constants in this file. But you can do that in a future PR if you want.

Reply:
Done for these 2. I can move the rest in a future PR